setup.py

```diff
@@ -22,20 +22,24 @@
         print("Value {} for USE_DISTUTILS treated as False".
               format(use_distutils))
 
-from distutils.command.build import build as _build
-
 if use_setuptools:
     try:
         from setuptools import setup
         from setuptools.command.install import install as _install
         from setuptools.command.build_ext import build_ext as _build_ext
     except ImportError:
         use_setuptools = False
+    else:
+        try:
+            from setuptools.command.build import build as _build
+        except ImportError:
+            from distutils.command.build import build as _build
```

Comment:
Why not just drop https://docs.python.org/3/library/distutils.html ?  It is history.

Reply:
Only recent builds of setuptools supports `setuptools.command.build`. See https://github.com/pypa/setuptools/pull/3256